### DISCUSSION TITLE
How to model multi meters?

### DISCUSSION THREAD

**Discussion by annieDehghani** [google/digitalbuildings#857] [Q&A]
How would you suggest modelling electrical panel-level meters that monitor multiple downstream equipment? 

Here's a minimal example to illustrate. 

Meter A has the following points: 
- Circuit 1 Equip. Power
- Circuit 1 Equip Energy
- Circuit 1 Equip Line Voltage
- Circuit 2 Equip. Power
- Circuit 2 Equip Energy
- Circuit 2 Equip Line Voltage
- Circuit 3 Equip. Power
- Circuit 3 Equip Energy
- Circuit 3 Equip Line Voltage

Should the individual circuits be modeled as their own meters?

**Comment by tasodorff** [comment]
hi annie, sorry that this response was so delayed: you can see here for how to monitor loads of specific types (e.g. main load): https://github.com/google/digitalbuildings/blob/master/ontology/docs/meter_loads.md

you can also use passthrough entities and remap each circuit to its own meter object, with a code of "circuit 1", "circuit 2", etc.
